lib/activerecord-import/import.rb

```diff
@@ -480,8 +480,8 @@ def import_without_validations_or_callbacks( column_names, array_of_attributes,
 
         # perform the inserts
         (number_inserted, ids) = connection.insert_many( [insert_sql, post_sql_statements].flatten,
-                                                  values_sql,
-                                                  "#{self.class.name} Create Many Without Validations Or Callbacks" )
+          values_sql,
+          "#{self.class.name} Create Many Without Validations Or Callbacks" )
```

Comment:
:thumbsup: for `with_fixed_indentation` over `with_first_parameter`. Can we turn off `with_first_parameter` as a viable option altogether? It's an abomination to all code that hopes to good in this world. :)